aliza/server.js

```diff
@@ -0,0 +1,43 @@
+'use strict';
+
+const http = require('http');
+
+const database = {};
+
+http.createServer((req, res) => {
+  if (req.url === '/time' && req.method === 'GET'){
+    let date = new Date();
+    let time = date.getHours() + ':' + date.getMinutes();
+    res.write(time + '\n');
+    return res.end();
+  }
+
+  let user = req.url.substring(7);
+  if (req.url === '/greet/' + user && req.method === 'GET'){
```

Comment:
I finally understand why you needed to declare user outside of the if statement.  However, since there might not always be a character at substring(7) like when the route is just '/greet', user will be undefined, but that's OK because it doesn't cause an error, it just silently returns "undefined".